libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/RouterWrapper.kt

```diff
@@ -92,6 +93,11 @@ class RouterWrapper(
                 },
                 {
                     mainJobControl.scope.launch {
+                        logI(
+                            "processing successful response " +
+                                "from router.getRoute for $urlWithoutToken",
```

Comment:
`urlWith**Token**` is not safety 🤔

Reply:
what's wrong with it? Is there anything secret other than token in the url?

Reply:
false alarm, I read `urlWithoutToken` like `urlWithToken` 🤦